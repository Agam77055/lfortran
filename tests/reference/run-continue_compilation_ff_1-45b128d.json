{
    "basename": "run-continue_compilation_ff_1-45b128d",
    "cmd": "lfortran --fixed-form --continue-compilation --no-color {infile}",
    "infile": "tests/errors/continue_compilation_ff_1.f",
    "infile_hash": "a516c3d2a0aa482137912f26a5baf756a859fbce7a0c6f3f623eb041",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "run-continue_compilation_ff_1-45b128d.stderr",
    "stderr_hash": "9cd33506bd4ae480fe9adc5f003a75580c22eb17b4d0b76ae74932b3",
    "returncode": 1
}