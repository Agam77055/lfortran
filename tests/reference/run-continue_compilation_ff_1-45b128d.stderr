semantic error: The shift argument of 'dshiftl' intrinsic must be less than or equal to the bit size of the integer
 --> tests/errors/continue_compilation_ff_1.f:3:30
  |
3 |       print *, dshiftl(1, 2, 34)
  |                              ^^ 

semantic error: The shift argument of 'dshiftl' intrinsic must be non-negative integer
 --> tests/errors/continue_compilation_ff_1.f:4:30
  |
4 |       print *, dshiftl(1, 2, -2)
  |                              ^^ 
